bluesky/__init__.py

```diff
@@ -16,3 +16,8 @@
 navdb = _navdb.navdb
 sim   = _sim.sim
 scr   = _scr.scr
+
+SIMPLE_ECHO = 'simple_echo'
+MSG_OK = "ok."
```

Comment:
Use single or double, I'd not switch between them. Their standard seems to be single which we shd stick with.